### DISCUSSION TITLE
[Feature Request] "Group by" modification time and other criteria

### DISCUSSION THREAD

**Discussion by jiucenglou** [doublecmd/doublecmd#1150] [Suggestions]
In Windows explorer, "Group by" modification time lists directories and files in invidual sections. The feature is quite convenient to locate files. Is it possible to achieve similar functionality in doublecmd ?

**Comment by WhiteChairFromIkea** [comment]
Isn't this sufficient: details view -> sort by "modified time" column. Scroll view down until needed modified time becomes visible. ..? (You can get any property shown (column) from files by going to preferences and changing column configuration there)

**Comment by WhiteChairFromIkea** [comment]
Also, what grouping criteria explorer is using? (Today, this week, this month ...?)

**Comment by WhiteChairFromIkea** [comment]
Also, one can make files and folders shown in different colors, depending on modified date. For instance, I see files and folders marked red if modified between 24 hours, and orange color if modified in two weeks (personal habits).

**Comment by jiucenglou** [comment]
> Scroll view down until needed modified time becomes visible.

As seen in the screen snapshot, the "group by" functionality makes it easier to locate files. :D It's helpful because it allows one to sort by one criterion while group by another one.

![image](https://user-images.githubusercontent.com/37147430/140652051-c20fbe5b-a123-4690-8ea3-13cba5a4e474.png)

**Comment by alexx2000** [comment]
You can use sorting by multiple columns: Left mouse click + Ctrl key.

**Comment by jiucenglou** [comment]
> Also, one can make files and folders shown in different colors, depending on modified date. For instance, I see files and folders marked red if modified between 24 hours, and orange color if modified in two weeks (personal habits).

Could you suggest how you apply colors based on modification time ?

**Comment by jiucenglou** [comment]
> You can use sorting by multiple columns: Left mouse click + Ctrl key.

Always something new to learn about doublecmd :D

**Comment by WhiteChairFromIkea** [comment]
> > Also, one can make files and folders shown in different colors, depending on modified date. For instance, I see files and folders marked red if modified between 24 hours, and orange color if modified in two weeks (personal habits).
> 
> Could you suggest how you apply colors based on modification time ?

I cannot find exact help page (found this functionality on the web), but configuration has something to do with these settings:    
![5110f](https://user-images.githubusercontent.com/46539653/140703862-30da00cb-430e-4aad-9e78-cd02833892fc.png)

![0001F](https://user-images.githubusercontent.com/46539653/140704947-844f4a88-376b-4501-8642-155933730b84.png)


I think you need to "Save search template", and later you can use that template in these settings. This is how Find dialog looks like in my configuration:    
![0325E](https://user-images.githubusercontent.com/46539653/140705210-e3362107-f4f1-4450-8d4a-8846f496459a.png)


This is how it looks like in my version of DC (red and orange files and folders):

![0038p](https://user-images.githubusercontent.com/46539653/140705117-a2b86487-c3c2-46fb-95a7-5a03c58da557.png)
